### PR TITLE
Add last activity column in the API tokens list

### PR DESCRIPTION
**Pull request by simotae14**
What does it do?
As an admin who has access to the API tokens settings, I want to know when the last operation on a token was executed, so that I can have basic information on the usage of every API token.

How to test it?
Should satisfy the following criterias:

- Given I’m an admin who has access to the API token settings

  - when I consult the list of the tokens

    - then I want the “Last used” column to be shown

  - and when I test an API token, I want the value of the 'Last used” field to be coherent with the time passed since I made the last call/test with that token.

### CODE REVIEW

**Comment by Convly** [comment]
Since I never touched this part of the codebase, I'm not sure it's relevant for me to review this PR :eyes:

File: packages/core/admin/package.json [hunk 1]
```diff
@@ -34,8 +34,7 @@
     "test:front": "cross-env IS_EE=true jest --config ./jest.config.front.js",
     "test:front:watch": "cross-env IS_EE=true jest --config ./jest.config.front.js --watchAll",
     "test:front:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js",
-    "test:front:watch:ce": "cross-env IS_EE=false jest --config ./jest.config.front.js --watchAll",
-    "test:front:ce:cov": "cross-env IS_EE=false jest --config ./jest.config.front.js --coverage --collectCoverageFrom='<rootDir>/packages/core/admin/admin/**/*.js' --coverageDirectory='<rootDir>/packages/core/admin/coverage'"
```
Comment (gu-stav) [inline packages/core/admin/package.json:38]: I think you don't want to commit that change.

File: packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/utils/tableHeaders.js [hunk 2]
```diff
@@ -16,18 +16,18 @@ const tableHeaders = [
...
     },
   },
   {
-    name: 'createdAt',
-    key: 'createdAt',
+    name: 'lastUsedAt',
+    key: 'lastUsedAt',
     metadatas: {
-      label: 'Created at',
+      label: 'Last used',
```
Comment (gu-stav) [inline packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/utils/tableHeaders.js:30]: @ronronscelestes Will merge this PR today which conflicts with these changes: https://github.com/strapi/strapi/pull/14111 and after this branch needs an update + merge. Could you have a look and take care of that?

**Review by gu-stav — CHANGES_REQUESTED** [review]
Added two comments: overall it looks good to me!

File: packages/core/admin/package.json (hunk 1 above)
Comment (simotae14) [reply packages/core/admin/package.json:38]: yes I are right

File: packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/utils/tableHeaders.js (hunk 2 above)
Comment (simotae14) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/utils/tableHeaders.js:30]: ok sure

File: packages/core/admin/package.json (hunk 1 above)
Comment (simotae14) [reply packages/core/admin/package.json]: done

File: packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/utils/tableHeaders.js (hunk 2 above)
Comment (simotae14) [reply packages/core/admin/admin/src/pages/SettingsPage/pages/ApiTokens/ListView/utils/tableHeaders.js]: done

**Comment by gu-stav** [comment]
@simotae14 There is a failing BE test. I don't know how this is related, but could you fix it before please 🙏🏼

**Review by Bassel17 — APPROVED** [review]

**Review by gu-stav — APPROVED** [review]

**Pull request merged**